test/load/README.md

```diff
@@ -0,0 +1,26 @@
+# Load Testing
+
+Load tests aim to test the performance of the system under heavy load. Game server allocation is an example where multiple parallel operations should be tested.
+
+## Build and run test
+
+Prerequisites:
+- Docker.
+- A running k8s cluster.
+- Have kubeconfig file ready.
+
+Load tests are written using Locust. These tests are also integrated with Graphite and Grafana
+for storage and visualization of the results. 
+
+To run load tests using Locust on your local machine:
+
+```
+docker build -t locust-files .
+docker run --rm --network="host" -e "TARGET_HOST=http://127.0.0.1:8001" locust-files:latest
+```
+
+The above will build a Docker container to install Locust, Grafana, and Graphite and will configure
+them. The test uses the HTTP proxy on the local machine to access the k8s API. 
```

Comment:
curious: is this not going to limit the perceived performance in some way (I would imagine proxy is probably not optimized for speed)?

Reply:
I'm not sure, it might. I will improve that in future iterations.